lib/routes/skeb/works.ts

```diff
@@ -0,0 +1,83 @@
+import { Data, DataItem, Route } from '@/types';
+import cache from '@/utils/cache';
+import ofetch from '@/utils/ofetch';
+import { config } from '@/config';
+import ConfigNotFoundError from '@/errors/types/config-not-found';
+import { processWork, baseUrl } from './utils';
+import InvalidParameterError from '@/errors/types/invalid-parameter';
+
+export const route: Route = {
+    path: '/works/:username',
+    categories: ['picture'],
+    example: '/works/@brm2_1925',
+    parameters: { username: 'Skeb Username with @' },
+    features: {
+        requireConfig: [
+            {
+                name: 'Request_Key',
+                optional: false,
+                description: '從 Cookie 中獲取',
+            },
+            {
+                name: 'Bearer_Token',
+                optional: false,
+                description: '在瀏覽器開發者工具（F12）的主控台中輸入 `localStorage.getItem("token")` 獲取',
+            },
+        ],
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: 'Creator Works',
+    maintainers: ['SnowAgar25'],
+    handler,
+    radar: [
+        {
+            title: 'Creator Works',
+            source: ['skeb.jp/:username'],
+            target: '/works/:username',
+        },
+    ],
+    description: 'Get the latest works of a specific creator on Skeb',
+};
+
+async function handler(ctx): Promise<Data> {
+    const username = ctx.req.param('username');
+
+    if (!username) {
+        throw new InvalidParameterError('Invalid username');
+    }
```

Comment:
This will always be skipped since https://github.com/DIYgod/RSSHub/blob/4028d798f71ecd280043b8f869f3cd5a25c9e2b0/lib/routes/skeb/works.ts#L10 defines `username` as a compulsory parameter.